config/webhook/manifests.yaml

```diff
@@ -137,7 +137,7 @@ webhooks:
       namespace: system
       path: /validate-opentelemetry-io-v1alpha1-opampbridge
   failurePolicy: Fail
-  name: vopampbridge.kb.io
+  name: vopampbridgecreateupdate.kb.io
```

Comment:
why did this name change?

Reply:
https://github.com/avadhut123pisal/opentelemetry-operator/blob/c3088a72c7bd0af9f7d3283875cfdb860cc309f2/apis/v1alpha1/opampbridge_webhook.go#L37C1-L37C1

The change has been done considering the validation webhook configurations of Collector resource. 
One webhook is for create & update operations (with failure policy as Fail) and another one is for delete operation (with failure policy as Ignore). Previously, by default there was a single webhook for all the operations.